### PR TITLE
fix: update docs, background health checks

### PR DESCRIPTION
**Pull request by jagadeeshi2i**
Signed-off-by: Jagadeesh J <jagadeeshj@ideas2it.com>

### CODE REVIEW

File: training-job/bert.slurm [hunk 1]
```diff
@@ -33,6 +33,10 @@ export LD_LIBRARY_PATH=/usr/local/lib/:$LD_LIBRARY_PATH
 # set the network interface
 export NCCL_SOCKET_IFNAME="en,eth,em,bond"
 
+
+# Uncomment for FSDP
+# ENABLE_NCCL_BASE_COLLECTIVES=0
```
Comment (chauhang) [inline training-job/bert.slurm:38]: Please add the same check for the Mnist FSDP example

File: training-job/install_pt.sh [hunk 2]
```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+export CUDA_HOME=/usr/local/cuda
+export NCCL_HOME=/home/ec2-user/packages/nccl/build
+export MPI_HOME=/opt/amazon/openmpi
+export LD_LIBRARY_PATH=${NCCL_HOME}/lib:${CUDA_HOME}/lib:${CUDA_HOME}/lib64:${CUDA_HOME}/extras/CUPTI/lib64:${MPI_HOME}/lib:${MPI_HOME}/lib64:/usr/local/lib:/usr/lib
```
Comment (chauhang) [inline training-job/install_pt.sh:6]: Where are the NCCL_HOME and CUDA_HOME environment variables getting set?

File: training-job/install_pt.sh [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    dataclasses -y
+conda install -c pytorch magma-cuda111 -y
+
+cd /lustre || exit
+git clone --recursive https://github.com/pytorch/pytorch
+cd pytorch || exit
+git checkout -b v1.11.0 v1.11.0
+git submodule sync
+git submodule update --init --recursive --jobs 0
+export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
```
Comment (chauhang) [inline training-job/install_pt.sh:23]: Where is the CONDA_PREFIX env variable getting set

Comment (jagadeeshi2i) [reply training-job/install_pt.sh:23]: conda sets that env while installing conda in the headnode post install script.

File: training-job/bert.slurm (hunk 1 above)
Comment (jagadeeshi2i) [reply training-job/bert.slurm:38]: added in PR #28

**Pull request merged**